build-system/test-configs/forbidden-terms.js

```diff
@@ -103,6 +103,79 @@ const forbiddenTermsGlobal = {
   'describes.*\\.only': {
     message: 'Please remove all instances of describes.only',
   },
+  // TODO(#35443) Reduce the number of files that use non default timeouts.
+  'this.timeout': {
+    message: 'Tests are not allowed to override the default timeout',
+    allowlist: [
+      // Markdown files.
+      'extensions/amp-a4a/rtc-documentation.md',
+      'extensions/amp-a4a/rtc-publisher-implementation-guide.md',
+
+      // Infrastructure
+      'build-system/tasks/e2e/describes-e2e.js',
+      'testing/describes.js',
+
+      // Thirdparty
+      'ads/google/a4a/test/test-utils.js',
```

Comment:
This is a test file, not a third_party file, so it can go into the next section.